lib/Base/CMakeLists.txt

```diff
@@ -17,7 +17,8 @@ if(PNG_FOUND)
                                ${PNG_INCLUDE_DIR})
   target_link_libraries(Base
                         PRIVATE
-                          ${PNG_LIBRARY})
+                          ${PNG_LIBRARY}
+                          z)
```

Comment:
Is this relevant/intentional?

Reply:
@jfix71 While compiling glow in docker(Based out of ubuntu 16.04) we have some times seen issues where libpng was giving error of missing zlib functions. For that reason "z" was added. If this is not required we can remove this line of change.

Reply:
🤷‍♂ Fine by me, just wanted to make sure it was intentional.